tests/modules/busco/main.nf

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { BUSCO as BUSCO_BACTE } from '../../../modules/busco/main.nf'
+
+// This tests genome decompression, empty input channels and data download
+workflow test_busco {
```

Comment:
Please provide tests suitable for offline usage too. I.e. when given a path to the lineage, it uses it successfully.